rmarkdowns/GISSR_W3_JakeFlanigan.Rmd

```diff
@@ -0,0 +1,105 @@
+```{r}
+install.packages("osmdata")
+```
+```{r}
+install.packages("leaflet")
+```
+
+```{r}
+library(tidyverse)
+```
+```{r}
+library(sf)
+```
+```{r}
+library(osmdata)
+``
+```
+```{r}
+library(leaflet)
+```
+```{r}
+available_features(
+  
+)
+```
+```{r}
+available_tags("landuse")
+```
+Challenge 1 above
+Challenge 2 below
+```{r}
+skiresorts <- opq (bbox = c (-125, 30, -100, 50)) %>% 
+    add_osm_feature (key = "landuse", value = "winter_sports", value_exact = FALSE) %>%
+    osmdata_sf ()
+```
+
+```{r}
+bb <- c (-125, 30, 100, 50)
+```
+
+```{r}
+bb <- c(-125, 30, 100, 50)
+```
+
+```{r}
+bb %>%
+  opq()
+```
+```{r}
+skiresorts <- bb %>%
+  opq() %>%
+  add_osm_feature(key = "landuse", value = "winter_sports") %>%
+  osmdata_sf()
+```
+
+```{r}
+skiresorts
+```
+```{r}
+skiresorts$metadata
+```
+
+```{r}
+skiresorts$osm_polygons
+```
+
+```{r}
+ggplot() +
+  geom_sf(data = skiresorts$osm_polygons) +
+  #geom_sf_text(data = skiresorts$osm_polygons) +
+  coord_sf(xlim = c(-125, -100), ylim = c(30, 50), expand = FALSE)
```

Comment:
I couldn't get it to pull in the data for the whole area for motorways, but here is the call to pull in cities:

```{r}
cities <- opq (bbox = c (-125, 32, -100, 49))%>%
  add_osm_feature("place", "city") %>%
  osmdata_sf()
```